app/helpers/application_helper.rb

```diff
@@ -16,6 +16,13 @@ def tooltip(text)
     )
   end
 
+  def alert_category(key)
+    key = key.to_s.downcase
+    return 'oops' if key.in? %w(error alert)
+    return 'success' if key.in? %w(notice success)
```

Comment:
+1 to @jessieay's comments. We favor the "Tell, Don't Ask" principle. This helper can be removed by standardizing the flash key names and creating matching CSS classes.